docs/docs/charts/bar.mdx

```diff
@@ -161,6 +161,7 @@ The bar chart accepts the following configuration from the associated dataset op
 | `barPercentage` | `number` | `0.9` | Percent (0-1) of the available width each bar should be within the category width. 1.0 will take the whole category width and put the bars right next to each other. [more...](#barpercentage-vs-categorypercentage)
 | `categoryPercentage` | `number` | `0.8` | Percent (0-1) of the available width each category should be within the sample width. [more...](#barpercentage-vs-categorypercentage)
 | `barThickness` | <code>number&#124;string</code> | | Manually set width of each bar in pixels. If set to `'flex'`, it computes "optimal" sample widths that globally arrange bars side by side. If not set (default), bars are equally sized based on the smallest interval. [more...](#barthickness)
+| `base` | `number` | | Base value for the bar in data units along the value axis. If not set, defaults to the value axis base value.
```

Comment:
I specified the type as `number`, but technically because all of these options go through the resolve process, all of these are scriptable and indexable (I tested that this worked).

Reply:
Should add a note about that?

Reply:
I can, what do you think about about two more columns like we have in the line docs? https://www.chartjs.org/docs/master/charts/line#dataset-properties

Reply:
Sounds good

Reply:
Realized there was a table with dataset properties at the top containing all the ones that are scriptable. I moved the definition of `base` up there.